docs/source/format/IPC.rst

```diff
@@ -24,13 +24,16 @@ Encapsulated message format
 Data components in the stream and file formats are represented as encapsulated
 *messages* consisting of:
 
+* A 32-bit continuation indicator (0xFFFFFFFF). This allows flatbuffer bytes
+  to begin on an 8-byte boundary.  This was added as of release 0.15.0.
 * A length prefix indicating the metadata size
```

Comment:
Change this to "A 32-bit length prefix"?
(also, where is endianness documented?)

Reply:
Changed.  Also clarified it is little endian.